src/grep.js

```diff
@@ -22,12 +24,15 @@ common.register('grep', _grep, {
 //@ + `-l`: Print only filenames of matching files.
 //@ + `-i`: Ignore case.
 //@ + `-n`: Print line numbers.
+//@ + `-B <num>`: Show `<num>` lines before each result.
+//@ + `-A <num>`: Show `<num>` lines after each result.
 //@
 //@ Examples:
 //@
 //@ ```javascript
 //@ grep('-v', 'GLOBAL_VARIABLE', '*.js');
 //@ grep('GLOBAL_VARIABLE', '*.js');
+//@ grep('-B', 3, 'GLOBAL_VARIABLE', '*.js');
```

Comment:
Can you please also document a case showing the object-style of passing the `-B` option?

Reply:
Done!